src/models/group.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+let Sequelize = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Consider putting the model under a separate variable and only then exporting it.

Reply:
resolved